src/main/kotlin/com/cognifide/gradle/aem/instance/satisfy/PackageGroup.kt

```diff
@@ -51,24 +52,32 @@ class PackageGroup(resolver: FileResolver, name: String) : FileGroup(resolver, n
      */
     var completer: () -> Unit = { await() }
 
-    private fun performAction(action: AbstractAction, closure: Closure<*>) {
-        action.configure(closure).perform()
+    fun await() {
+        await({})
     }
 
-    fun await(closure: Closure<*>) {
-        AwaitAction(project, instances).configure(closure).perform()
+    fun await(configurer: Closure<*>) {
+        await({ ConfigureUtil.configure(configurer, this) })
     }
 
-    fun await() {
-        AwaitAction(project, instances).perform()
+    fun await(configurer: AwaitAction.() -> Unit) {
+        action(AwaitAction(project, instances), configurer)
     }
 
-    fun reload(closure: Closure<*>) {
-        ReloadAction(project, instances).configure(closure).perform()
+    fun reload() {
+        reload({})
     }
 
-    fun reload() {
-        ReloadAction(project, instances).perform()
+    fun reload(configurer: Closure<*>) {
+        reload({ ConfigureUtil.configure(configurer, this) })
```

Comment:
such construction allows to make plugin already compatible with both Groovy and Kotlin based DSL